src/__tests__/index.test.ts

```diff
@@ -11,6 +11,7 @@ const formData: FormData = {
     "name": 'testName',
 };
 
+let uid = 'testUserId';
```

Comment:
Parang wala ako nakitang assertion na nagamit itong uid na pinasa sa initClient?

Reply:
Na test mo na  ba sa test na ung pinasa na uid sa initClient is yun ung path na ni writan nya ng form?